interactive.c

```diff
@@ -77,9 +79,40 @@ static int convert_speed(int kts)
 // Show the currently captured interactive data on screen.
 //
 
+double distance_units_conversion;
+char  *distance_units_suffix;
+regex_t callsign_filter_regex;
+
 void interactiveInit() {
-    if (!Modes.interactive)
+    if (!Modes.interactive) {
         return;
+    }
+
+    switch(Modes.interactive_distance_units) {
+    case UNIT_NAUTICAL_MILES:
+        distance_units_conversion = 0.53996;
+        distance_units_suffix = "nm";
+        break;
+    case UNIT_STATUTE_MILES:
+        distance_units_conversion = 0.621371;
+        distance_units_suffix = "sm";
+        break;
+    case UNIT_KILOMETERS:
+        distance_units_conversion = 1.0;
+        distance_units_suffix = "km";
+        break;
+    }
+
+    if (Modes.interactive_callsign_filter) {
+        int rc = regcomp(&callsign_filter_regex, Modes.interactive_callsign_filter,
+            REG_EXTENDED | REG_NOSUB);
```

Comment:
REG_ICASE probably makes sense here too

Reply:
Yep.  I thought about that, then forgot. :)